spec/index.bs

```diff
@@ -80,53 +119,134 @@ interface GPUBufferUsage {
     const u32 STORAGE   = 0x0080;
     const u32 INDIRECT  = 0x0100;
 };
+</script>
 
-dictionary GPUBufferDescriptor : GPUObjectDescriptorBase {
-    required u64 size;
-    required GPUBufferUsageFlags usage;
-};
+## Buffer Mapping ## {#buffer-mapping}
 
-interface GPUBuffer : GPUObjectBase {
-    Promise<ArrayBuffer> mapReadAsync();
-    Promise<ArrayBuffer> mapWriteAsync();
-    void unmap();
+<script type=idl>
+typedef sequence<any> GPUMappedBuffer;
+</script>
+
+{{GPUMappedBuffer}} must be a sequence of 2 elements of types {{GPUBuffer}} and
+{{ArrayBuffer}}, respectively. Otherwise, an exception is thrown.
```

Comment:
`GPUMappedBuffer` is a type returned by the implementation, not an argument passed by the application, so I don't think we need to specify the exception bit.

Reply:
Oh, right. Thanks!